lib/connection/apm.js

```diff
@@ -200,7 +201,8 @@ class CommandSucceededEvent {
       commandName,
       reply: maybeRedact(commandName, extractReply(command, reply)),
       requestId: command.requestId,
-      connectionId: generateConnectionId(pool)
+      connectionId: generateConnectionId(pool),
+      stepsData: new CommandStepsData(stepsData)
```

Comment:
I'd just name the field `metadata` here, so we could use it for other metadata items.